src/Compilers/CSharp/Portable/Symbols/FunctionPointerTypeSymbol.cs

```diff
@@ -0,0 +1,264 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Reflection;
+using Microsoft.Cci;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal class FunctionPointerTypeSymbol : TypeSymbol
+    {
+
+        public static FunctionPointerTypeSymbol CreateFunctionPointerTypeSymbolFromSource(FunctionPointerTypeSyntax syntax, Binder typeBinder, DiagnosticBag diagnostics, ConsList<TypeSymbol> basesBeingResolved)
+        {
+            var symbol = new FunctionPointerTypeSymbol();
+            var signature = FunctionPointerMethodSymbol.CreateMethodFromSource(symbol, syntax, typeBinder, diagnostics, basesBeingResolved);
+
+            symbol.Signature = signature;
+
+            return symbol;
+        }
+
+        public static CallingConvention GetCallingConvention(string convention) =>
+            convention switch
+            {
+                "" => CallingConvention.Default,
+                "cdecl" => CallingConvention.C,
```

Comment:
>C [](start = 45, length = 1)

It looks like there is a Cdecl member

Reply:
At least there is one in System.​Runtime.​Interop​Services.CallingConvention. Perhaps we should add this alias into our type so that we could use language terminology throughout.

---
In reply to: [359063427](https://github.com/dotnet/roslyn/pull/40431#discussion_r359063427) [](ancestors = 359063427)